docs/build/web.md

```diff
@@ -96,7 +103,9 @@ To get all build artifacts of ONNX Runtime WebAssembly, it needs 4 times of buil
 
 ### Minimal Build Support
 
-ONNX Runtime WebAssembly can be built with flag `--minimal_build`. This will generate smaller artifacts and also have a less runtime memory usage. An ORT format model is required. A detailed instruction will come soon. See also [ORT format Conversion](../reference/ort-format-models.md).
+ONNX Runtime WebAssembly can be built with flag `--minimal_build`. This will generate smaller artifacts and also have a less runtime memory usage. 
+In order to use this ONNX Runtime confiruation an ORT format model is required (vs. ONNX format). 
+A detailed guide will come soon. for more info please see also [ORT format Conversion](../reference/ort-format-models.md).
```

Comment:
IMO no need to say "A detailed guide will come soon." the link should be enough

Reply:
Totally agree, I am removing it